include/swift/AST/DiagnosticsSIL.def

```diff
@@ -856,5 +856,12 @@ NOTE(sil_referencebinding_inout_binding_here, none,
       "inout binding here",
       ())
 
+// Warnings arising from the flow-sensitive checking of Sendability of
+// non-Sendable values
+WARNING(consumed_value_used, none,
+        "Non-Sendable value consumed, then used at this site; could yield race with another thread", ())
+WARNING(arg_region_consumed, none,
+        "This application could pass `self` or a Non-Sendable argument of this function to another thread, potentially yielding a race with the caller", ())
```

Comment:
small note: we usually don't capitalize the first letter of diagnostics unlesss its proper noun. Also would be good to avoid the "consume" terminology since we use that for move-only types. Perhaps we can call these actions "transfers" of non-sendable values that are causing the uses afterwards to be illegal.